src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -726,7 +726,37 @@ internal override void Complete()
 
                     this.Command.Context.SetVariable(SpecialVariables.LastExitCodeVarPath, _nativeProcess.ExitCode);
                     if (_nativeProcess.ExitCode != 0)
```

Comment:
@rjmholt The code after this code is basically just for handling exceptions (and finally cleanup).  I think this code would read easier (less indentation) if I switched up to early return instead of nesting. Thoughts?

Reply:
Absolutely agreed. I want to hold fewer states in my head as I read down.

Reply:
Done